toolkits/global/packages/global-forms/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@springernature/global-forms",
-	"version": "0.0.2",
+	"version": "0.2.0",
```

Comment:
In the history you have it as 0.1.0

Reply:
Good spot, that was silly. I'm holding off on this for now anyway.